metadata/v1.6/step_1_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.6",
+  "description": "This includes changes outlined in https://github.com/aptos-labs/aptos-core/releases/aptos-node-v1.6",
```

Comment:
this is not a valid url, you need to use 1.6.0

Reply:
This is all generated by the builder. Let me fix

Reply:
Updated